Automattic/woocommerce-payments
# Fix styling for enabled currencies components, cleaned up classnames.

Fixes #1920

#### Changes proposed in this Pull Request

* Updates enabled currencies settings components to match styling of initial design.
* Clean up classnames and other markup.

#### Testing instructions

* Switch to the branch.
* Go to WooCommerce > Settings > Multi-Currency.
* Check that the Enabled Currencies list displays correctly.
* Click the Add Currencies button and check that the modal items display correctly.
* Click the Delete/Trash icon and and check that the model displays similar to the other items.

-------------------
